pkg/ingester/instance.go

```diff
@@ -300,6 +306,7 @@ func (i *instance) createStreamByFP(ls labels.Labels, fp model.Fingerprint) *str
 
 	i.streamsCreatedTotal.Inc()
 	memoryStreams.WithLabelValues(i.instanceID).Inc()
+	memoryStreamsLabels.WithLabelValues(i.instanceID).Add(float64(len(s.labels.String())))
```

Comment:
`labels.Labels` has a `Len()` function which you should use instead I believe.

Reply:
Actually `Len()` of `labels.Labels` just return total number of elements in `labels.Labels` slice. We need a byte size of all the labels.

```
type Labels []Label

func (ls Labels) Len() int           { return len(ls) }

```

Reply:
Oops!